jfrog-cli/utils/tests/utils.go

```diff
@@ -37,6 +37,8 @@ var TestGo *bool
 var DockerRepoDomain *string
 var DockerTargetRepo *string
 var TestNuget *bool
+var MaskCredentials *bool
```

Comment:
I think that the *MaskCredentials* option is redundant. Let's always mask the credentials.